canonical/ubuntu.com
# WD-12057 dev rebrand /cloud/public-cloud

## Done

-  Rebranded /cloudpublic-cloud
## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
-  [Design](https://www.figma.com/design/NpI2wzW86XXPWxsSeomYfy/24.10-u.c%2Fcloud?node-id=182-957&t=DFnFIxrcFDnDvRuH-0)
-  [Copydoc](https://docs.google.com/document/d/1ZzUIh9Tfz5SvkrHyo4-izzmUjRka1Tn0UQisKcXGEPA/edit#heading=h.6lhihgdcnq6v)
-  [Demo]()

## Issue / Card

Fixes # [WD-12057](https://warthogs.atlassian.net/browse/WD-12057)
## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-12057]: https://warthogs.atlassian.net/browse/WD-12057?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
